app/scripts/controllers/pod.js

```diff
@@ -87,6 +87,49 @@ angular.module('openshiftConsole')
       }
     };
 
+    /**
+     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
+     */
+    $scope.onTerminalSelectChange = function(newTerm) {
+      // Deactivate all terminals
+      _.each($scope.containerTerminals, function(term) {
+        term.isActive = false;
+      });
+
```

Comment:
@benjaminapetersen onTerminalSelectChange receives {key, value} from the view, hence the need to use .value to get at the actual data.